android/Tests/Chirality/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.keyman.chirality" >
```

Comment:
can we use a name such as `com.keyman.android.tests.chirality`? Avoid polluting the top-level namespace with all the various tests we may have in future.